taxi/views.py

```diff
@@ -58,8 +60,26 @@ class CarListView(LoginRequiredMixin, generic.ListView):
     queryset = Car.objects.all().select_related("manufacturer")
 
 
-class CarDetailView(LoginRequiredMixin, generic.DetailView):
-    model = Car
+# class CarDetailView(LoginRequiredMixin, generic.DetailView):
+#     model = Car
+
+@login_required()
+def car_detail_view(request: HttpRequest, pk: int) -> HttpResponse:
+    car = Car.objects.prefetch_related("drivers").filter(pk=pk).first()
```

Comment:
It's better to use get_object_or_404 for fetching the car object instead of filter(...).first(). This will handle the case where the car does not exist more gracefully.